src/ferryctl/cmd/root.go

```diff
@@ -17,13 +17,16 @@
 package cmd
 
 import (
+	"os"
+	"path"
+
 	"github.com/spf13/cobra"
 )
 
 // RootCmd is the main entry point into ferry
 var RootCmd = &cobra.Command{
-	Use:   "ferry",
-	Short: "ferry is the Solus package repository tool",
+	Use:   path.Base(os.Args[0]),
+	Short: path.Base(os.Args[0]) + " is the Solus package repository tool",
```

Comment:
What does this change get us?

Reply:
If you launch `ferryctl --help`, the help string is hardcoded to suggest to launch `ferry`. That's wrong, since this is `ferryctl`. The new code prints the name of the executable no matter what it is.